File: ABC/175/a_ans.c
#include <stdio.h>
#include <string.h>

int main(){
  // input
  char S[3]; scanf("%s", S);

  // compute
  char R = 'R';
  int p = !strcmp(S[0], R);
  int q = !strcmp(S[1], R);
  int r = !strcmp(S[2], R);
   // output
  if (p && q && r){
    printf("3\n");
  }
  else if ((p&&q) || (q&&r)){
    printf("2\n");
  }
  else if (p || q || r){
    printf("1\n");
  }
  else{
    printf("0\n");
  }
  return 0;
}

// Received "Error code: 139"...

File: AGC/029/a_ans.py
def main():
    # input
    S = input()

    # compute
    ans, cnt = 0, 0
    for s in S[::-1]:
        if s == 'W':
            cnt += 1
        else:
            ans += cnt

    # output
    print(ans)


if __name__ == '__main__':
    main()
